src/collections/quote_list.js

```diff
@@ -3,6 +3,13 @@ import Quote from 'models/quote';
 
 const QuoteList = Backbone.Collection.extend({
   model: Quote,
+
+  bySymbol: function(symbol) {
+    const filtered = this.filter(function (quote) {
+      return quote.get('symbol') === symbol;
```

Comment:
I like that you broke this out as a separate function on the collection. Good organization!